pkg/instancetype/instancetype.go

```diff
@@ -22,16 +22,26 @@ import (
 	generatedscheme "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/scheme"
 	"kubevirt.io/client-go/kubecli"
 	"kubevirt.io/client-go/log"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 
 	utils "kubevirt.io/kubevirt/pkg/util"
 	utiltypes "kubevirt.io/kubevirt/pkg/util/types"
 )
 
+const (
+	DefaultInstancetypeAnnotation     = "instancetype.kubevirt.io/defaultInstancetype"
```

Comment:
Isn't the convention in K8S to use kebab-case instead of camel-case?

e.g. `instancetype.kubevirt.io/default-instancetype`

Reply:
I can't find any documented preference for this and we have both styles in the codebase :

```
$ git grep Annotation\ string staging/src/kubevirt.io/api/core/v1/types.go 
staging/src/kubevirt.io/api/core/v1/types.go:   DomainAnnotation string = "kubevirt.io/domain"
staging/src/kubevirt.io/api/core/v1/types.go:   ControllerAPIStorageVersionObservedAnnotation string = "kubevirt.io/storage-observed-api-version"
staging/src/kubevirt.io/api/core/v1/types.go:   FuncTestForceLauncherMigrationFailureAnnotation string = "kubevirt.io/func-test-force-launcher-migration-failure"
staging/src/kubevirt.io/api/core/v1/types.go:   FuncTestBlockLauncherPrepareMigrationTargetAnnotation string = "kubevirt.io/func-test-block-migration-target-preparation"
staging/src/kubevirt.io/api/core/v1/types.go:   FuncTestMigrationTargetImageOverrideAnnotation string = "kubevirt.io/func-test-migration-target-image-override"
staging/src/kubevirt.io/api/core/v1/types.go:   FuncTestLauncherFailFastAnnotation string = "kubevirt.io/func-test-virt-launcher-fail-fast"
staging/src/kubevirt.io/api/core/v1/types.go:   FuncTestForceIgnoreMigrationBackoffAnnotation string = "kubevirt.io/func-test-ignore-migration-backoff"
staging/src/kubevirt.io/api/core/v1/types.go:   EvacuationMigrationAnnotation string = "kubevirt.io/evacuationMigration"
staging/src/kubevirt.io/api/core/v1/types.go:   WorkloadUpdateMigrationAnnotation string = "kubevirt.io/workloadUpdateMigration"
staging/src/kubevirt.io/api/core/v1/types.go:   KeepLauncherAfterFailureAnnotation string = "kubevirt.io/keep-launcher-alive-after-failure"
staging/src/kubevirt.io/api/core/v1/types.go:   MigrationTransportUnixAnnotation string = "kubevirt.io/migrationTransportUnix"
staging/src/kubevirt.io/api/core/v1/types.go:   MigrationUnschedulablePodTimeoutSecondsAnnotation string = "kubevirt.io/migrationUnschedulablePodTimeoutSeconds"
staging/src/kubevirt.io/api/core/v1/types.go:   MigrationPendingPodTimeoutSecondsAnnotation string = "kubevirt.io/migrationPendingPodTimeoutSeconds"
staging/src/kubevirt.io/api/core/v1/types.go:   CustomLibvirtLogFiltersAnnotation string = "kubevirt.io/libvirt-log-filters"
staging/src/kubevirt.io/api/core/v1/types.go:   InstancetypeAnnotation string = "kubevirt.io/instancetype-name"
staging/src/kubevirt.io/api/core/v1/types.go:   ClusterInstancetypeAnnotation string = "kubevirt.io/cluster-instancetype-name"
staging/src/kubevirt.io/api/core/v1/types.go:   PreferenceAnnotation string = "kubevirt.io/preference-name"
staging/src/kubevirt.io/api/core/v1/types.go:   ClusterPreferenceAnnotation string = "kubevirt.io/cluster-preference-name"
staging/src/kubevirt.io/api/core/v1/types.go:   PVCMemoryDumpAnnotation string = "kubevirt.io/memory-dump"
```

Reply:
^ made me realise this constant also needs to move into the instance type API, I'll sort that out once we decide which convention to go with.

Reply:
Other instancetype related annotations are alreadu using kebab-case, so I think we should stick with it.

Reply:
ACK, switched and moved to the instancetype API.